service/src/main/java/org/databiosphere/workspacedataservice/pubsub/JobStatusUpdate.java

```diff
@@ -12,12 +13,16 @@ public JobStatusUpdate(UUID jobId, StatusEnum currentStatus, StatusEnum newStatu
   }
 
   public static JobStatusUpdate createFromPubSubMessage(PubSubMessage message) {
-    Map<String, String> attributes = message.attributes();
-    UUID jobId = UUID.fromString(attributes.get("import_id"));
-    StatusEnum newStatus = rawlsStatusToJobStatus(attributes.get("new_status"));
-    StatusEnum currentStatus = rawlsStatusToJobStatus(attributes.get("current_status"));
-    String errorMessage = attributes.get("error_message");
-    return new JobStatusUpdate(jobId, currentStatus, newStatus, errorMessage);
+    try {
+      Map<String, String> attributes = message.attributes();
+      UUID jobId = UUID.fromString(attributes.get("import_id"));
+      StatusEnum newStatus = rawlsStatusToJobStatus(attributes.get("new_status"));
+      StatusEnum currentStatus = rawlsStatusToJobStatus(attributes.get("current_status"));
+      String errorMessage = attributes.get("error_message");
+      return new JobStatusUpdate(jobId, currentStatus, newStatus, errorMessage);
+    } catch (Exception e) {
+      throw new ValidationException("Unable to parse job status update from PubSub message: %s", e);
```

Comment:
did you mean to put something in the `%s` here?
```suggestion
      throw new ValidationException(
          "Unable to parse job status update from PubSub message: %s".formatted(e.getMessage()), e);
```